utam-core/src/main/java/utam/core/framework/element/BasePageElement.java

```diff
@@ -171,7 +171,8 @@ public void click() {
       getElement().click();
     } catch (Exception e) {
       if (e.getMessage()
-          .contains("javascript error: Cannot read property 'defaultView' of undefined")) {
+          .contains("javascript error: Cannot read") && e.getMessage()
+              .contains("defaultView")) {
```

Comment:
Add another clause to the condition: `&& e.getMessage().contains("undefined")`.

Reply:
Added condition  to check "undefined" error